species.rb

```diff
@@ -0,0 +1,12 @@
+require 'byebug'
+
+module Species
+extend self
```

Comment:
I think this should be indented, so two extra spaces at the beginning

Reply:
Thanks!